templates/layouts/roq-default/post.html

```diff
@@ -19,6 +19,14 @@
 	padding-top: 0.3rem;
 	padding-bottom: 0.3rem;
 }
+.wrapper {
+  max-width: 1500px;
+}
+pre code.hljs {
+  max-width: 1016px;
+  white-space: pre;
+  background-color: #222;
+}
```

Comment:
The width change is messing up with images alignments
![image](https://github.com/user-attachments/assets/71c2b556-e7e1-4449-9a9d-fd00fc754325)

Reply:
```suggestion
.wrapper {
  max-width: 1500px;
}
pre code.hljs {
  max-width: 1016px;
  white-space: pre;
  background-color: #222;
}
p:has(>img) {
    text-align: center;
}
```

Reply:
thanks!